tcp_ip:
  ip_addr_server: "tcp://*"
  ip_addr_client: "tcp://192.168.0.30"
  interface_name: "wlan0"
  req_port: "4444"
  rep_port: "7777"
  zipcode: "00000"

tcpimg_ip:
  ip_addr_server: "tcp://*"
  ip_addr_client: "tcp://192.168.0.11"
  req_port: "50000"
  rep_port: ""

udp_ip:
  ip_addr: "udp://239.255.255.251"
  port: "9091"
  send_group: "FV"
  recv_group: "" 

socket:
  rad_flag: false
  dsh_flag: false
  req_flag: false 
  rep_flag: true
  req_img_flag: true
  rep_img_flag: false

params:
  index: 0
  target_vel: 0.00
  safety_vel: 0.45
  fv_max_vel: 1.20
  rcm_vel: 0.6
  lv_stop_dist: 0.5
  fv_stop_dist: 0.3
  safety_dist: 1.5
  target_dist: 0.8
  rcm_dist: 0.8
  angle_degree: 0.0
  Kp_d: 2.0
  Ki_d: 0.2
  K: 0.15
  a:
    a: 0.
    b: -0.37169
    c: 1.2602
    d: -1.5161
    e: 0.70696
  b:
    a: 0.
    b: -1.7536
    c: 5.0931
    d: -4.9047
    e: 1.6722

Calibration:
  f_matrix:
    a: 329.10805843346054
    b: 0.
    c: 320. 
    d: 0. 
    e: 329.10805843346054
    f: 240. 
    g: 0. 
    h: 0. 
    i: 1.
  f_dist_coef:
    a: -0.34335984453339752
    b: 0.14676511736197059
    c: 0.
    d: 0. 
    e: -0.032260115806245951
  r_matrix:
    a: 326.31389227574556
    b: 0.
    c: 320. 
    d: 0. 
    e: 326.31389227574556
    f: 240. 
    g: 0. 
    h: 0. 
    i: 1.
  r_dist_coef:
    a: -0.33295846454356126
    b: 0.12386827336557986
    c: 0.
    d: 0. 
    e: -0.022565312043601477

ROI:
  front_cam:
    width: 640
    height: 480
    top_gap: 0.416
    bot_gap: 0.185
    top_height: 0.99
    bot_height: 0.47
    extra_f: 0.00
    extra_b: 0.00
    extra_up: 140
    extra_down: 180
  rear_cam:
    width: 640
    height: 480
    top_gap: 0.405
    bot_gap: 0.17
    top_height: 0.99
    bot_height: 0.47
    extra_f: 1
    extra_b: 10
    extra_up: 140
    extra_down: 180

rear_camera_usage:
  use: true
